src/show/components/Button/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+
+const Button = ({
+  type = 'button',
```

Comment:
see mui buttons or bootstrap button , where you can see type like 'Outline', Filled and others

Reply:
sir I saw that but i did not get exactly how to do here.